src/frontend/Komponenter/Behandling/Brev/Brev.tsx

```diff
@@ -110,10 +136,11 @@ const Brev: React.FC<Props> = ({ behandlingId }) => {
                         </HøyreKolonne>
                     </StyledBrev>
                     <SendTilBeslutterFooter
-                        behandlingId={behandlingId}
+                        behandling={behandling}
                         kanSendesTilBeslutter={kanSendesTilBeslutter}
                         ferdigstillUtenBeslutter={skalFerdigstilleUtenBeslutter(vedtak)}
                         behandlingErRedigerbar={behandlingErRedigerbar}
+                        oppgaverForOpprettelseState={oppgaverForOpprettelseState}
```

Comment:
Navngivning: er det nødvendig å ha "state" i navnet? Tror det kunne vært mer forklarende å kalle dette `oppgaverSomSkalOpprettes`

Hvorfor trenger man å sende med `oppgaverSomKanOpprettes?`

Reply:
Tja, kanskje man ikke trenger en state suffiks, selv om det er en state ? Litt usikker. Vi må ha med oppgaverSomKanOpprettes for å kunne se hvilke oppgaver som det skal genereres valg for (akkurat nå finnes det bare en oppgave som kan opprettes). Vi trenger oppgaverSomSkalOpprettes for å se hvilke oppgaver det har blitt tatt stilling til. Ikke kjempeintuitivt. Skal jeg fjerne state suffiksen ?